huaweicloud/services/lts/resource_huaweicloud_lts_cce_access.go

```diff
@@ -0,0 +1,530 @@
+// @API LTS POST /v3/{project_id}/lts/access-config
+// @API LTS PUT /v3/{project_id}/lts/access-config
+// @API LTS POST /v3/{project_id}/lts/access-config-list
+func ResourceCceAccessConfig() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: resourceCceAccessConfigCreate,
+		UpdateContext: resourceCceAccessConfigUpdate,
+		ReadContext:   resourceCceAccessConfigRead,
+		DeleteContext: resourceHostAccessConfigDelete,
+		Importer: &schema.ResourceImporter{
+			StateContext: cceAccessConfigResourceImportState,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"region": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+				ForceNew: true,
+			},
+			"name": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+				Description: `The name of the CCE access. The name consists of 1 to 64 characters.
+ Only letters, digits, underscores (_), and periods (.) are allowed, and the period cannot be the first
+ or last character.`,
+			},
+			"log_group_id": {
+				Type:        schema.TypeString,
+				Required:    true,
+				ForceNew:    true,
+				Description: `The log group ID.`,
+			},
+			"log_stream_id": {
+				Type:        schema.TypeString,
+				Required:    true,
+				ForceNew:    true,
+				Description: `The log stream ID.`,
+			},
+			"access_config": {
+				Type:        schema.TypeList,
+				MaxItems:    1,
+				Required:    true,
+				Elem:        cceAccessConfigDeatilSchema("access_config.0"),
+				Description: `The configurations of CCE access.`,
+			},
+			"cluster_id": {
+				Type:        schema.TypeString,
+				Required:    true,
+				ForceNew:    true,
+				Description: `The CCE cluster ID.`,
+			},
+			"host_group_ids": {
+				Type:        schema.TypeList,
+				Elem:        &schema.Schema{Type: schema.TypeString},
+				Optional:    true,
+				Computed:    true,
+				Description: `The log access host group ID list.`,
+			},
+			"tags": common.TagsSchema(),
+			"binary_collect": {
+				Type:        schema.TypeBool,
+				Optional:    true,
+				Computed:    true,
+				Description: `The binary collection switch. Default is false.`,
+			},
+			"log_split": {
+				Type:        schema.TypeBool,
+				Optional:    true,
+				Computed:    true,
+				Description: `The log split switch. Default is false.`,
+			},
+			"access_type": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: `The log access type.`,
+			},
+			"log_group_name": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: `The log group name.`,
+			},
+			"log_stream_name": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: `The log stream name.`,
+			},
+			"created_at": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: `The creation time of the CCE access, in RFC3339 format.`,
+			},
+		},
+	}
+}
+
+func cceAccessConfigDeatilSchema(parent string) *schema.Resource {
```

Comment:
```suggestion
func cceAccessConfigDeatilSchema() *schema.Resource {
```
just one place to use this function.

Reply:
done